plugin/trino-delta-lake/pom.xml

```diff
@@ -167,6 +167,12 @@
         <dependency>
             <groupId>org.apache.parquet</groupId>
             <artifactId>parquet-format-structures</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>javax.annotation</groupId>
+                    <artifactId>javax.annotation-api</artifactId>
```

Comment:
Why? Is the exclusion avoidabe?

Reply:
It's not. It's a transitive dependency used by Parquet. We are on jakarta and javax is a banned one.